src/main/java/model/TaskList.java

```diff
@@ -0,0 +1,63 @@
+package main.java.model;
+
+import java.util.*;
```

Comment:
Do not import everything with `*`. You can change the import settings in Intellij to prevent it from automatically optimising (Preferences > Editor > Code Style > Java, change the count to use import with `*` to a high number).